File: python/testTileset.py
# coding=UTF-8
import os, sys, pygame
import formats.tileset as tileset
import renderers.tilesetRenderer as renderer


# Tell the tileset format loader which path to use for loading tilesets:
tileset.tilePath = os.path.realpath('../testFiles/tiles/') + '/'

# Load the tileset file:
testBoard = tileset.load('RPGMaker951.tst')

# Initialise pygame:
pygame.init()

# Render the tileset:
renderer.render(testBoard)

# Pygame event loop:
clock = pygame.time.Clock()

while True:
	clock.tick(60)
	for event in pygame.event.get(): 
		if event.type == pygame.QUIT: 
			sys.exit(0)

File: php/testTileset.php
<?php

require_once('./binaryio.php');
require_once('./formats/tileset.php');
require_once('./renderers/tilesetRenderer.php');

Tileset::$tilePath = '../testFiles/tiles/';
$tileset = Tileset::load('RPGMaker951.tst');

$renderer = new TilesetRenderer($tileset);

#header('Content-Type: text/plain');
header('Content-Type: image/jpeg');
print $renderer->render('jpeg');